src/lib/helpers/ManifestHelper.ts

```
@@ -0,0 +1,96 @@
+      },
+      {
+        src: `${origin}/web-app-manifest-512x512.png`,
+        sizes: '512x512',
+        type: 'image/png',
+        purpose: 'any maskable',
+      },
+    ],
+  };
+}
+
+export function updateManifestLink() {
```

ESLint

[error] Delete `␍`

(prettier/prettier)